### DISCUSSION TITLE
BSBLAN Arduino DUE, freezing

### DISCUSSION THREAD

**Discussion by proasnet** [fredlcore/BSB-LAN#670] [General]
Dear friends.
I have BSB LAN, Version 4.0.57-20240721015110.
Sometimes it happens that it freezes. It happens about once or twice a month. It happened to me with older versions as well, and I don't know the reason.
Is it possible to insert a watchdog into the program for DUE, which restarts the Arduino DUE when it freezes?
Thank you

**Comment by fredlcore** [comment]
Unfortunately, the Arduino Due doesn't support watchdogs against freezing like the ESP32. The only thing you can do is to keep logging and figure out if the freezing has the same reason each time, then we could find and hopefully fix the bug. Otherwise, I don't really see a possibility here...
